spec/api/v2/api_bookmarks_spec.rb

```diff
@@ -0,0 +1,173 @@
+require "spec_helper"
+require "api/api_helper"
+
+describe "API V2 BookmarksController" do
+  include ApiHelper
+
+  bookmark = { id: "123",
+               url: "http://foo.com",
```

Comment:
Should we use `example.com`? I think that's our preferred sample domain for tests.